src/lib/settings.js

```diff
@@ -526,26 +529,156 @@ export function getDefaultSession() {
   return result;
 }
 
-function buildConfigurationMap201() {
-  const query = getDocumentQuery();
-  const list = configurationList201;
-  const result = {};
+class VariableConfiguration201 {
+  constructor(variables) {
+    this.variables = variables.reduce((acc, item) => {
+      const key = getConfigurationKey201(item);
+      acc[key] = item;
+      return acc;
+    }, {});
+  }
 
-  for (const item of list) {
-    const key = getConfigurationKey201(item);
-    const value = getConfigurationValue201(item);
+  getOCPPIdentityString() {
+    return this.getVariableActualValue('SecurityCtrlr.Identity');
+  }
 
-    if (query.get(key)) {
-      result[key] = query.get(key);
-      continue;
+  getMeterValueSampleInterval() {
+    const defaultInterval = 60;
+    const value = this.getVariableActualValue('MeterValueSampleInterval');
+    return value || defaultInterval;
+  }
+
+  variablesToSimpleConfigurationMap() {
```

Comment:
I'm seeing this `simple` in a few different places. I wonder if we should call something more straightforward to make it easier to understand what simple means. (eg: `variablesToKeyValueMap`)

name is just a suggestion, you can choose another one.

Reply:
Good one, was struggling with this as well. The one you suggested might be nice actually.